fix: removing references to (previous) versions of Che in two .adoc files (#2062)
codenvy/che-docs: modules/end-user-guide/examples/che-ref_a-kubernetes-secret-as-an-variable.adoc

    app.kubernetes.io/component: workspace-secret
...
----

// The corresponding procedure is underdeveloped. Parts of this example will likely end up in the procedure. A comprehensive rewrite is needed. max-cx

> * The following example shows you how to mount the secret as an environment variable by including the `<url>/mount-as: env` annotation.

Reviewer: ```suggestion
* The following example shows how to mount the secret as an environment variable by including the `<url>/mount-as: env` annotation:
```

    app.kubernetes.io/component: workspace-secret
...
----

> The following examples show more `metadata` of a secret:

Reviewer 2: ```suggestion
The following examples show additional `metadata` elements you can use with a secret:
```

* The following example shows you how to mount the secret as an environment variable by including the `<url>/mount-as: env` annotation.

* The secret's environment variable (`/env-name:`) is declared (`FOO_ENV`) in the `annotations`.
* The `/automount-workspace-secret` annotation with a boolean value defines the default secret mounting behavior (that can be overridden in a devfile):

** `true` enables automatic mounting into all workspace containers.

** `false` disables the mounting process until it is explicitly requested in a devfile component using the `automountWorkspaceSecrets:true` property.

> * In the next example, the environment variable `FOO_ENV` and the value `myvalue` will be provisioned into all workspace containers.

Reviewer 2: ```suggestion
* The following example provisions the `FOO_ENV` environment variable and `myvalue` value into all workspace containers.
```

Reviewer 2: Avoid future tense and passive voice.

Reviewer 2: Use commas to indicate parenthetical remarks: `the environment variable, 'FOO_ENV', and...` 
Or, avoid parenthetical remarks altogether: `the 'FOO_ENV' environment variable and..."

Author: Yes, "the 'FOO_ENV' environment variable" is the best option here. Thank you.

  mykey: myvalue
----
+
In this example, the `FOO_ENV` environment variable and `myvalue` key value are provisioned into all workspace containers as follows:

** The secret's environment variable, `env-name`, is declared as `FOO_ENV` in the `annotations`.

** The `automount-workspace-secret` annotation with a boolean value defines the default secret mounting behavior (that can be overridden in a devfile):

> *** `'true'` enables automatic mounting into all workspace containers.

Reviewer: Do these two values need to be in quotation marks?

Author: They have single quotes in the example code blocks provided here.

data:
  mykey: myvalue
----

> * If the secret provides more than one data key, each key must have its own environment variable name declared in the `annotations`. In the next example, the two environment variables `FOO_ENV` and `OTHER_ENV` and the two `data` values `myvalue` and `othervalue` will be provisioned into all workspace containers.

Reviewer 2: ```suggestion
* If the secret has more than one data key, declare an environment variable name for each key in the `annotations`, as shown in the following example. In this example, you provision the `myvalue` and `othervalue` data values, and the environment values they reference, `FOO_ENV` and `OTHER_ENV`, into all workspace containers.
```

Author: I wonder if there's a dependency, and as a result also a convention, like so:
`<_mykey_>_env-name` and `<_mykey_>`
`<_otherkey_>_env-name` and `<_otherkey_>`

Author: What about this?

If the secret has more than one data key, declare an environment variable name using a `<_key_>_env-name` annotation for each key, as shown in the following example. In this example, you provision the `myvalue` and `othervalue` data key values, and the environment variable names, `FOO_ENV` and `OTHER_ENV`, into all workspace containers.

  mykey: myvalue
  otherkey: othervalue
----

* The maximum length of an annotation name in a {orch-name} secret is 63 characters, of which 9 characters are reserved for a prefix that ends with `/`. This acts as a restriction for the maximum length of the key that can be used for the secret.

> //"annotation name" is referring to `name` or `env-name`? This is not clear from the code snippets presented in the code blocks here and so needs to be clarified. max-cx

Reviewer: Please, clarify this and modify the content accordingly.

Author: @rkratky I've saved this comment in a new topic branch for the follow-on issue (3166), so I will work on it there. The lack of clarity is in the original text, and I did not change its meaning; hence, it can be left in the text as is for now. I have also posted this question in #forum-devtools-docs and will ask someone again later. So here I am simply removing this comment line from this PR, which can be merged then.
